overlord/patch/export_test.go

```diff
@@ -42,7 +42,24 @@ func MockLevel(lv, sublvl int) (restorer func()) {
 	Level = lv
 	oldSublvl := Sublevel
 	Sublevel = sublvl
+	oldPatches := make(map[int][]PatchFunc)
```

Comment:
I'm not sure I understand why we need this change?

Reply:
This is because with addition of new sublevel patch I discovered this mocking helper doesn't mock things enough; it needs to drop excess sublevel patches as otherwise the test for patch level 6.1 also runs 6.2 patch.